src/mds/CInode.h

```diff
@@ -1050,6 +1050,15 @@ class CInode : public MDSCacheObject, public InodeStoreBase, public Counter<CIno
                            MDSContext *fin);
   static void dump_validation_results(const validated_data& results,
                                       ceph::Formatter *f);
+  bool has_inline_data() {
+    if (is_normal() && is_file()) {
+      auto mi = get_inode();
```

Comment:
Shouldn't we use the projected inode here ?

Reply:
done